packages/runtime/container-runtime/src/runningSummarizer.ts

```diff
@@ -93,15 +96,18 @@ export class RunningSummarizer implements IDisposable {
         firstAck: ISummaryAttempt,
         private readonly raiseSummarizingError: (description: string) => void,
         private readonly summaryCollection: SummaryCollection,
+        { disableHeuristics = false }: Readonly<Partial<ISummarizerOptions>> = {},
     ) {
         this.logger = ChildLogger.create(
             baseLogger, "Running", { all: { summaryGenTag: () => this.generator.getSummarizeCount() } });
 
-        this.heuristics = new SummarizerHeuristics(
-            configuration,
-            (reason) => this.trySummarize(reason),
-            lastOpSeqNumber,
-            firstAck);
+        this.heuristics = disableHeuristics
```

Comment:
I am curious as to why just not have heuristics when its disabled? That would make it more obvious that we are not summarizing unless there is an explicit summarize attempt (summarizeOnDemand).

Reply:
Please look again after my last push, I changed it to no longer have the runner. I still need the data tracker.